tools/commissioner_thci/commissioner_impl.py

```diff
@@ -431,9 +434,7 @@ def _sleep(self, seconds):
     def _read_line(self):
         logging.debug('Reading line')
         if len(self._lines) > 1:
-            line = self._lines.pop(0)
-            line = CONTROL_SEQUENCE.sub('', line)
-            return line
+            return CONTROL_SEQUENCE.sub('', self._lines.pop(0))
```

Comment:
nit: According to how `_lines` are used in code, [`deque`](https://docs.python.org/2/library/collections.html#deque-objects) is the better choice for data structure?

Reply:
Agree, but consider `list` is simpler and performance is not critical to our usage, I will keep this.